### PR TITLE
Add videoUrl field

### PR DESCRIPTION
**Pull request by beaucarnes**
Add vidoeUrl to responsive web design challenges.

### CODE REVIEW

File: challenges/01-responsive-web-design/applied-visual-design.json [hunk 1]
```diff
@@ -488,6 +494,7 @@
       "releasedOn": "Feb 17, 2017",
       "challengeType": 0,
       "translations": {},
+   c/Url": "",
```
Comment (Bouncey) [inline challenges/01-responsive-web-design/applied-visual-design.json]: Here is your linting error

Comment (Bouncey) [inline challenges/01-responsive-web-design/applied-visual-design.json]: Here is your linting error

**Review by Bouncey — CHANGES_REQUESTED** [review]

**Comment by Bouncey** [comment]
@beaucarnes You have some linting errors, also you will need to update the challengeSchema to allow the videoUrl property to be a string, Joi may allow you to check a vaild url also? If it takes more than a minute of looking up I wouldn't worry about it too much.

File: schema/challengeSchema.js [hunk 2]
```diff
@@ -29,6 +29,7 @@ const schema = Joi.object().keys({
     })
   ),
   guideUrl: Joi.string().uri({ scheme: 'https' }),
+  videoUrl: Joi.string(),
```
Comment (Bouncey) [inline schema/challengeSchema.js]: To allow for empty strings, this will need to be 
```js
Joi.string().allow('')
```

**Review by Bouncey — CHANGES_REQUESTED** [review]
There have been some changes to the schema merged recently, you will need to rebase for those changes.

File: schema/challengeSchema.js (hunk 2 above)
Comment (Bouncey) [reply schema/challengeSchema.js]: CI failed on `videoUrl: ''`

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
Nice work. Thanks @beaucarnes

**Pull request merged**